service/src/main/java/bio/terra/pipelines/db/entities/PipelineInputDefinition.java

```diff
@@ -18,10 +18,10 @@
 @SuppressWarnings("java:S107") // Disable "Methods should not have too many parameters"
 public class PipelineInputDefinition extends BasePipelineVariableDefinition {
   @Column(name = "is_required", nullable = false)
-  private Boolean isRequired;
+  private boolean isRequired;
 
   @Column(name = "user_provided", nullable = false)
-  private Boolean userProvided;
+  private boolean userProvided;
 
   @Column(name = "default_value")
   private String defaultValue; // must be a String representation of the value
```

Comment:
@jsotobroad do you think we should remove the default value for any custom inputs (i.e. refPanelPathPrefix)? either remove or put in "OVERRIDDEN BY CONFIG" as the default value or something?

Reply:
i would rather have the value it has now over "OVERRIDEN BY CONFIG" and i would rather have no value than the other options.  If we do remove the value lets make sure we add it to the different environments (including bees/qa) before we merge.

Reply:
yeah regardless we need to add to GSM before merge (and that's noted in the PR description, though not as a checklist item).

Reply:
i lied, i thought it was in the PR description, but it's only in the [terra-helmfile PR description](https://github.com/broadinstitute/terra-helmfile/pull/6014). adding to checklist!